### PR TITLE
Add a new copy of protos in cirq.google.api

### PR DESCRIPTION
**Pull request by dstrain115**
- We will eventually get rid of cirq.api.google.
- This will coalesce our two api.google directories together.

### CODE REVIEW

**Comment by maffoo** [comment]
Ref: #1873. In that issue, you mentioned that these copies will have to coexist until everyone has switched over to the new package name. Do we have a place to track all the changes that will need to happen?

File: cirq/google/api/v1/__init__.py [hunk 1]
```diff
@@ -13,6 +13,11 @@
 # limitations under the License.
 """Data format v1 for google api."""
 
+from cirq.api.google.v1 import (
```
Comment (dabacon) [inline cirq/google/api/v1/__init__.py]: was this mean to not be google.api?

**Comment by dabacon** [comment]
Do we think this will take long enough that we should add a test that just compares the files in case someone updates one and not other?

**Comment by dstrain115** [comment]
> Do we think this will take long enough that we should add a test that just compares the files in case someone updates one and not other?

I really hope that it doesn't last that long that we need this.  A few weeks hopefully?  I will try to move things as quick as possible.  Can you and Craig just keep an eye out for changes to these files for now?

**Pull request merged**